# This message is a container for a Turtlebot's MMA-generated name, and its Odometry topic.
string name
nav_msgs/Odometry odom